webapp/src/modules/components/ui-custom/breadcrumbs.ts

```diff
@@ -25,7 +25,7 @@ export async function calcBreadcrumbs(
 ): Promise<Link[]> {
 	const { exclude = [], renamers = {} } = options;
 
-	const routeChunks = getPathChunkList(page.route.id!);
+	const routeChunks = getPathChunkList(page.url.pathname!);
```

Comment:
@seabassdk On a second thought, I'm not sure this is fine

`getPathChunkList` actually requires the route.id, since subsequent transformations look for square brackets inside the route (that are missing inside the pathname).

Options:
- to close this quick, make one new ad-hoc component for the breadcrumbs specific for the markdown pages
- otherwise, we need to leave page.route.id, and edit the code in order to make it work with this scenario